backend/mms_app_backend/mms_app_backend/api/mentors/route.py

```diff
@@ -22,8 +24,18 @@ async def get_mentors(response: Response, db: Session = Depends(get_db), jwt_tok
         return get_mentors_response
 
     mentors = get_mentors_crud(db)
+    if mentors:
```

Comment:
@Albert-Byrone  @Munalula-Sikazwe, one of you removed this in previous changes.

Reply:
I changed the code because it did not handle  no mentors on the system.